src/Inter/builtins.rs

```
@@ -207,51 +207,51 @@ fn builtin_ucase(args: &[Value]) -> Result<Option<Value>, CPSError> {
-//                                                                     // <- makes it really useless
-//     if args.len() != 1 {
-//         return Err(arg_count_error("INT", 1, args.len()));
-//     }
-//     let real = expect_real(&args[0], "INT", 1)?;
-//     let result = real.floor() as i64;
-//     Ok(Some(Value::Integer(result)))
-// }
-
-
-fn builtin_int(args: &[Value]) -> Result<Option<Value>, CPSError> {
+fn builtin_int(args: &[Value]) -> Result<Option<Value>, CPSError> { 
```

Clippy (1.97.1)

[warning] the `Err`-variant returned from this function is very large

(warning)